lib/commands/appstore-upload.ts

```diff
@@ -34,6 +34,8 @@ export class PublishIOS implements ICommand {
 			mobileProvisionIdentifier = args[2],
 			codeSignIdentity = args[3],
 			teamID = this.$options.teamId,
+			exportMethod = this.$options.exportMethod ? this.$options.exportMethod : null,
```

Comment:
lib/commands/appstore-upload.ts(37,33): error TS2339: Property 'exportMethod' does not exist on type 'IOptions'.
lib/commands/appstore-upload.ts(37,62): error TS2339: Property 'exportMethod' does not exist on type 'IOptions'.
lib/commands/appstore-upload.ts(38,27): error TS2339: Property 'xcargs' does not exist on type 'IOptions'.
lib/commands/appstore-upload.ts(38,50): error TS2339: Property 'xcargs' does not exist on type 'IOptions'.

from travis: please fix